gov-pay-client/build.gradle

```diff
@@ -1,5 +1,32 @@
 description = ''
+configurations.all {
```

Comment:
As discussed & shown please keep the config in Root Build.gradle [all projects section]